### ISSUE TITLE
WCAG 2.1.1 Ensure User Can Linearly Navigate the Site with Tabs (Home Page)

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#576]
_From @xilin95 on October 29, 2018 19:36_

Linked to Issue #683 and #833 

**Info on tabindex:** https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Review the whole site to ensure tab sequence is correct and focus is placed at the right element at the right time. User should be able to reach all interactive controls by tabbing**

**Guidelines for Manage Focus:**
- Know that keyboard-only users navigate by moving focus thru the DOM using the Tab key
- Proactively place focus using element.focus() to create an efficient workflow 
- When changing tabs, entering modal, etc., make sure focus is put in an appropriate element in the new view. When that view is closed, return focus to where it was before. 
- Make custom interactive elements focusable with _tabindex_

**Some areas where the tab sequence needs fixing:**
- map
- guided assistant

**Check out the Google Accessibility video: https://www.youtube.com/watch?v=x18vEEfpK3g (18:00 mark)**
**Angular keyboard navigation:** https://stackoverflow.com/questions/37019509/angularjs-keyboard-navigation-using-tab-key-in-different-groups-forms-divs

_Copied from original issue: Microsoft/Access2Justice#703_
